tools/wavetool.py

```diff
@@ -112,6 +115,8 @@ def do_wave_analysis():
         analyze_wav_smart_amp(wave, fs_wav)
     if cmd.analyze == 'wov':
         analyze_wav_wov(wave, fs_wav)
```

Comment:
I think we should use `if elif` here

Reply:
`parser.add_argument('-a', '--analyze', type=str, choices=['smart_amp', 'wov', 'thdn'] ...`. argparse will guarantee us when code goes here, cmd.analyze is one of `['smart_amp', 'wov', 'thdn'] `, so with or without using `if elif` should not matter. if we give something else to cmd.analyze, argparse will exit the program and dump help info.